build-esmf-docs/esmf/Dockerfile

```diff
@@ -32,8 +32,9 @@ RUN make
 # Collect and compress the documentation artifacts
 WORKDIR ${ESMF_DIR}
 RUN cp -rv doc ${ESMF_ARTIFACTS}/doc-esmf
+RUN cp -v src/doc/*dvi ${ESMF_ARTIFACTS}/doc-esmf
```

Comment:
Do you mean
`RUN cp -v src/doc/*.dvi ${ESMF_ARTIFACTS}/doc-esmf`

Reply:
Good idea to store them in a place where they do not get published. I'll make that change.